lib/macho/macho_file.rb

```diff
@@ -424,14 +478,24 @@ def low_fileoff
       offset
     end
 
+    # Updates the number of load commands in the raw data.
+    # @param ncmds [Fixnum] the new number of commands
+    # @return [void]
+    # @private
+    def set_ncmds(ncmds)
```

Comment:
How do you feel about naming this `update_ncmds` or `ncmds=` to make it a true setter method? The reason I'm asking is that naming methods `set_*` in Ruby is somewhat atypical.

Another thing I'm wondering about is whether `set_ncmds` and `set_sizeofcmds` should be merged to a `update_header` method that always updates both (and possibly defaults to the previous number of commands, if omitted). It's just that these two values are very closely related and updating them together would slightly simplify the packing and related business. (But that's just a thought. I can definitely see an argument for keeping them separate.)